<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V2Ray-Qt5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/icons/v2ray.png</normaloff>:/icons/icons/v2ray.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="filterEdit">
      <property name="placeholderText">
       <string>Input to filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="ServersTableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Import_Connections_from_gui_config_json"/>
    <addaction name="actionExport_as_gui_config_json"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <widget class="QMenu" name="menu_Add">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <addaction name="actionManually"/>
     <addaction name="actionScan_QR_Code"/>
    </widget>
    <addaction name="menu_Add"/>
    <addaction name="action_Edit"/>
    <addaction name="action_Share"/>
    <addaction name="action_Delete"/>
    <addaction name="separator"/>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="separator"/>
    <addaction name="actionMove_Up"/>
    <addaction name="action_Move_Down"/>
   </widget>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="action_General_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionBug_Report"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Connection"/>
   <addaction name="menu_Setting"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_Connect"/>
   <addaction name="action_Disconnect"/>
   <addaction name="separator"/>
   <addaction name="action_Edit"/>
   <addaction name="action_Delete"/>
   <addaction name="action_Share"/>
   <addaction name="separator"/>
   <addaction name="actionMove_Up"/>
   <addaction name="action_Move_Down"/>
  </widget>
  <action name="action_Import_Connections_from_gui_config_json">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/import-icon.png</normaloff>:/icons/icons/import-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Connections from json file</string>
   </property>
  </action>
  <action name="actionExport_as_gui_config_json">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/export-icon.png</normaloff>:/icons/icons/export-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export as json file</string>
   </property>
   <property name="iconText">
    <string>Export as json file</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/save-icon.png</normaloff>:/icons/icons/save-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/exit-icon.png</normaloff>:/icons/icons/exit-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/edit-icon.png</normaloff>:/icons/icons/edit-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="action_Share">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/config-share-icon.png</normaloff>:/icons/icons/config-share-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Share</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/config-delete-icon.png</normaloff>:/icons/icons/config-delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/connect-icon.png</normaloff>:/icons/icons/connect-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/disconnect-icon.png</normaloff>:/icons/icons/disconnect-icon.png</iconset>
   </property>
   <property name="text">
    <string>D&amp;isconnect</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/up-icon.png</normaloff>:/icons/icons/up-icon.png</iconset>
   </property>
   <property name="text">
    <string>Move &amp;Up</string>
   </property>
  </action>
  <action name="action_Move_Down">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/down-icon.png</normaloff>:/icons/icons/down-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Move Down</string>
   </property>
  </action>
  <action name="action_General_Settings">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/setting-icon.png</normaloff>:/icons/icons/setting-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;General Settings</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/about-icon.png</normaloff>:/icons/icons/about-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/qt-icon.png</normaloff>:/icons/icons/qt-icon.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionBug_Report">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/report-icon.png</normaloff>:/icons/icons/report-icon.png</iconset>
   </property>
   <property name="text">
    <string>Bug &amp;Report</string>
   </property>
  </action>
  <action name="actionManually">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/manually-icon.png</normaloff>:/icons/icons/manually-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manually</string>
   </property>
  </action>
  <action name="actionScan_QR_Code">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/qrcode-icon.png</normaloff>:/icons/icons/qrcode-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scan QR Code</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
